spec/models/financial_transaction_spec.rb

```diff
@@ -57,4 +57,23 @@
       expect(incidental_trans.amount).to eq(incidental.incidental_type.base+(incidental.times_modified*incidental.incidental_type.modifier_amount))
     end
   end
+
+  describe '#self.get_rental_summation' do
+    before(:each) do
+      @item_type = create :item_type, name: 'TEST_ITEM_TYPE'
+      @rental = create :rental, item_type: @item_type
+      @incidental = create :incurred_incidental, rental: @rental
+    end
```

Comment:
The [rental factory](https://github.com/umts/probable-engine/blob/master/spec/factories/rental.rb#L5) has an [association](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#associations) that creates an `ItemType` automagically.

Further, since you are not using `@incidental`, you can create the rental in line

Becomes something to the effect of:
```Ruby
before(:each) do
  @rental = create :rental
  create :incurred_incidental, rental: @rental
end
```

If you were feeling as though you wanted to prune out an amazing amount of code, refactoring the [incurred_incidental factory](https://github.com/umts/probable-engine/blob/master/spec/factories/incurred_incidental.rb) to eliminate the mock object could result in this section of code becoming:
```Ruby
before(:each) do
  @rental = create(:incurred_incidental).rental
end
```